lib/Doctrine/Migrations/Configuration/AbstractFileConfiguration.php

```diff
@@ -23,6 +23,7 @@
         'migrations_namespace',
         'table_name',
         'column_name',
+        'created_at_column_name',
```

Comment:
Maybe `executed_at_column_name`, and consequently, a rename of the corresponding methods, would be more appropriate?

After all, this reflects the time of when the migration was executed, rather then when the migration (file) was created.

What do you think?